rskj-core/src/main/java/co/rsk/core/bc/BlockRelayValidatorImpl.java

```diff
@@ -58,6 +62,7 @@ public BlockRelayValidatorImpl(@Nonnull BlockStore blockStore,
     @Override
     public boolean isValid(@Nonnull Block block) {
         if (block.isGenesis()) {
+            logger.error("Block is Genesis");
```

Comment:
`logger.warn()` ?

Reply:
I decided to use error because I think this shouldn't be ever be happening nowadays, receiving a Genesis block at this stage, but yeah I can change it.